src/components/summary/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {Label} from '../common/label/label.js';
+
+import {FilmSort} from '../summary/film-sort/film-sort';
+
+import PropTypes from 'prop-types';
+
+const Summary = (props) => (
+  <div className='summary'>
+    <div className='summary-wrapper'>
+      {props.filmMode? null : <Label content ={'For \''+ props.searchPhrase +'\' ' + props.filmNumber + ' movies found'} />}
+      {props.filmMode? null : <FilmSort sortItems={props.sortItems} changeSortItemCallBack = {props.changeSortItemCallBack} />}
```

Comment:
Can be done like this
` {!props.filmMode && <FilmSort sortItems={props.sortItems} changeSortItemCallBack = {props.changeSortItemCallBack} />}`